packages/proppy-redux/src/withStore.ts

```diff
@@ -50,6 +50,17 @@ export function withStore(
       }
     },
 
+    handleReceivedProps(parentProps) {
+      if (mapState) {
+        this.set(parentProps);
```

Comment:
some tests would be lovely that can validate the need for having two `this.set()`s.

Reply:
Will need to figure out how to do that, but sure thing :)

Reply:
@fahad19 I've added a test, test fails if the first `this.set()` is removed.